common/lib/xmodule/xmodule/course_module.py

```diff
@@ -294,6 +294,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the url of the official Facebook page of the course."),
+        default="",
+        display_name=_("Course Facebook Page"),
+        scope=Scope.settings
+    )
```

Comment:
@lou-wang , will the navigation to Facebook be available in the mobile app for all users? I thought it was only for those users who are get the app through the Rwanda/Facebook/Zero-rating project? perhaps I misunderstood when we discussed this earlier.